Fluid/Filters/MiscFilters.cs

```diff
@@ -614,14 +614,24 @@ private static async ValueTask WriteJson(Utf8JsonWriter writer, FluidValue input
                                 value = access.Get(obj, name, ctx);
                             }
 
+                            if (value is object)
```

Comment:
It's always true unless value is null

Reply:
This was to check for null since this is the value returned from the property accessor, not the object itself.  I had originally put that there when the `.Add()` was in that block as I didn't want to add a null value to the `stack`.

Since I move the `.Add()` to after the `fluidValue.IsNil()`, removing it as you suggested makes sense.  Also, just to verify I'm understanding the flow properly, the `obj` object that gets added to the `stack` will always be a non-primitive, non-string/date/built-in, type, correct?  Otherwise, the `FluidValue` that is passed to `WriteJson` would trigger a different case statement.  Is my logic correct there?

Thanks for the review!

Reply:
All good